components/elm/src/dyn_subgrid/dynSubgridAdjustmentsMod.F90

```diff
@@ -876,10 +885,12 @@ subroutine dyn_col_ns_Adjustments(bounds, clump_index, column_state_updater, col
            var         = sminn_vr(begc:endc, j), &
            adjustment  = adjustment_one_level(begc:endc))
 
-       col_ns%dyn_nbal_adjustments(begc:endc) = &
-           col_ns%dyn_nbal_adjustments(begc:endc) + &
-           adjustment_one_level(begc:endc) * dzsoi_decomp(j)
-
+       ! TRS NaN trap, for debugging
```

Comment:
1. The comments say "for debugging"...are these NaN traps still needed? If not, remove; If yes, remove the "for debugging" language
2. Remove Tim Shippert's initials; not needed and potentially confusing

Reply:
I have replaced these checks with simple if statements, where the code would exit if it detects a NaN.  That is, we won't skip any calculations based on NaN detection; the code will exit and let us know that it has detected NaNs. As we all know, NaNs floating around in a simulation can cause all kinds of headaches!